cassiopeia/data/CassiopeiaTree.py

```diff
@@ -962,7 +963,7 @@ def set_character_states(
         if self.is_leaf(node):
             if self.get_character_states(node) == []:
                 raise CassiopeiaTreeError(
-                    "Leaf node character states have not been instantiated"
+                    "Character states have not been initialized at leaves."
```

Comment:
Can you tell people how to initialize character states at leaves in this error message?